test/unittests/state_transition_eof_create_test.cpp

```diff
@@ -69,6 +91,29 @@ TEST_F(state_transition, create2_with_eof_initcode)
     expect.post[*tx.to].storage[0x01_bytes32] = 0x01_bytes32;
 }
 
+TEST_F(state_transition, create2_with_eof_initcode_cancun)
+{
+    rev = EVMC_CANCUN;
+    block.gas_limit = 10'000'000;
+    tx.gas_limit = block.gas_limit;
+    pre.get(tx.sender).balance = tx.gas_limit * tx.max_gas_price + tx.value + 1;
+
+    const bytecode init_container = eof_bytecode(ret(0, 1));
```

Comment:
we usually use `eof_bytecode(OP_INVALID)` as the smallest valid eof container. Also, `eof_bytecode(ret(0, 1))` fails validation due to wrong max stack height. However irrelevant to the logic being tested, I think it's safer to use a valid EOF code here.

Ah, I see the same is used above. I think this should be fixed there as well when we're at it.